src/RedisProxyFactory.php

```diff
@@ -56,6 +91,7 @@ public function createFromConfig(array $config): RedisProxy
             );
         }
 
+        /** @phpstan-ignore-next-line  Unreachable by documented config union, but kept for runtime safety */
```

Comment:
neznasam tieto inline komentare, to v kode nema co robit, phpstan je uplne oddelena vec od kodu, a ma svoj phpstan.neon file kde sa chyby ignoruju